### PR TITLE
Rename UTxOState fields without underscores

### PR DESCRIPTION
**Pull request by aniketd**
Partially addresses #3061

- [x] Rename fields across current code base
~~- [ ] Add DEPRECATED warning for all renamed fields~~ They won't be useful as setters.
~~- [ ] Open PRs against downstream packages to not use the deprecated accessors.~~ The CHANGELOG update could be enough.

### CODE REVIEW

**Review by JaredCorduan — APPROVED** [review]
looks wonderful to me, thank you @aniketd !

File: eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs [hunk 1]
```diff
@@ -158,7 +158,7 @@ instance
         ( \(TRC (LedgerEnv {ledgerPp}, _, _))
            (LedgerState utxoSt DPState {dpsDState, dpsPState}) ->
               obligation ledgerPp (rewards dpsDState) (_pParams dpsPState)
-                == _deposited utxoSt
+                == utxosDeposited utxoSt
```
Comment (teodanciu) [inline eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs:161]: Just curious, is it `utxos` as in the plural of `utxo` or from `utxoState`  ?

**Review by lehins — CHANGES_REQUESTED** [review]
This is a breaking change and requires a bump in version. So, currently released version of `cardano-ledger-shelley` is `0.1.0.0`, so we need to bump it up to `0.2.0.0`. Otherwise it looks great. This is a fairly new process, which we should all adopt

Also, we need to add a changelog to all of our packages and start placing changes there, instead of top level CHNAGELOG file. @aniketd This could be your next task, which you could do in a follow up PR

File: eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs (hunk 1 above)
Comment (aniketd) [reply eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs:161]: :+1:  It stands for `UTxO{S}tate` as in `State`, and not for plurality. :slightly_smiling_face:

**Comment by teodanciu** [comment]
Should we also [release the new version  to cHaP](https://github.com/input-output-hk/cardano-haskell-packages#how-to-add-a-new-package-version-to-chap)?

**Comment by lehins** [comment]
No, we don't need to make a release for every PR. Only when a downstream library is ready to integrate and use the new version, we should make a release

**Review by lehins — APPROVED** [review]
Looks great. It just needs a conflict resolution and I suggest to squash all commits into one

**Comment by aniketd** [comment]
Done!

**Review by teodanciu — APPROVED** [review]
Looks good to me

**Review by Soupstraw — APPROVED** [review]
LGTM

**Comment by teodanciu** [comment]
@aniketd  - this is the most highly approved PR I've seen!  :smile:  :tada:

**Pull request merged**